library/Notifications/Widget/ItemList/ContactGroupListItem.php

```diff
@@ -23,7 +25,9 @@ class ContactGroupListItem extends BaseListItem
 
     protected function init(): void
     {
-        $this->getAttributes()->set('data-action-item', true);
+        $this->getAttributes()
+            ->set('data-action-item', true)
+            ->set('data-icinga-detail-filter', QueryString::render(Filter::equal('id', $this->item->id)));
```

Comment:
> We should stay consistent and use Icinga\Module\Notifications\Common\Links for path references to strengthen the maintainability.

I'd extend this to query param references:

`Links::contactGroup($this->item->id)->getQueryString()`